LICENCE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
You mean changing the `licenCe` to `licenSe` only right? Because as for the UPPERCASE notation it seems be a standard for such license files 
https://github.com/kyma-project/kyma/blob/main/LICENSE 
https://github.com/kyma-project/infrastructure-manager/blob/main/LICENCE
https://github.com/kyma-project/compass-manager/blob/main/LICENSE
https://github.com/kyma-project/btp-manager/blob/main/LICENSE

Reply:
[2c7be31](https://github.com/kyma-project/application-connector-manager/pull/165/commits/2c7be31362cb8e8438d694d0a1d1b944384a308c)

Reply:
Yes, yes. That's what I meant :) Thanks!